catalog/ui/search-ui/standard/src/main/webapp/js/view/ingest/IngestModal.view.js

```diff
@@ -36,9 +36,6 @@ define([
                 'click .finish-upload': 'finishUpload'
             },
             initialize: function () {
-                // there is no automatic chaining of initialize.
-                Modal.prototype.initialize.apply(this, arguments);
```

Comment:
So this still just broke all of the other modals. I think there is a better way to do this and still keep the other ones working. The main issue is that hiding the modal causes the view to be destroyed because a listener is being attached. What you really need to do is update Modal.js so that a modal can pass in its own function for destroying the view and if the modal doesn't pass in its own function then it just uses the default one it has now. Then that function that you pass in can actually take into account whether or not the view is done doing its thing. If it isn't, you don't destroy it.